runner.go

```diff
@@ -65,10 +65,15 @@ func main() {
 
 	http.HandleFunc("/ws/", handlers.NewWSConn)
 
-	fmt.Println("Binding to address: " + *addr)
-	err = http.ListenAndServe(*addr, nil)
-	utils.FailOnError(err, "Could not bind to port")
+	addr := fmt.Sprintf("0.0.0.0:%d", cfg.ServerConfig.Port)
+	utils.LogInfo("Starting server", utils.LogFields{
+		"Address": addr,
+	})
+	err = http.ListenAndServe(addr, nil)
+	utils.LogError("Could not bind to port", err, nil)
 
 	// Kill the SetupRabbitExchange thread (Multithreading control)
-	AMQPControl.Exit <- true
+	defer func() {
+		AMQPControl.Exit <- true
+	}()
```

Comment:
Deferred calls also happen on panic and non-standard function exits, so it's still preferred as much as possible.